.github/SECURITY.md

```diff
@@ -0,0 +1,32 @@
+# Responsible Security Disclosure README standard
+
+## Our security policy and Your responsibility
+
+- **POLICY**:
+
+*Our security policy is to avoid leaving the ecosystem worse than we found it. Meaning we are not planning to introduce vulnerabilities into the ecosystem.*
+
+The "ITGlueAPI" team and community take all security bugs in "ITGlueAPI" seriously. Thank you for improving the security of "ITGlueAPI". We appreciate your efforts and responsible disclosure and will make every effort to acknowledge your contributions.
+
+Report security bugs by emailing the lead maintainer at [placeholder@example.com](placeholder@example.com) and include the word "SECURITY" in the subject line..
```

Comment:
Another email address placeholder and bonus period at end of line

Reply:
oops, that'll be an easy fix, and ya another placeholder from a canned security readme